about.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="./assets/images/logo.png">
+    <link rel="icon" href="./assets/images/logo.png">
+    <title>About</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <header class=" fixed-top bg-light">
+    <div id="social-media-nav" class="d-none d-md-flex bg-dark justify-content-end align-items-center">
+      <a href="https://www.facebook.com/MotorBikeExpoOfficial/" class="" target="_blank" rel="noopener noreferrer">
+        <img class="img-ratio mx-3" src="./assets/icons/facebook-icon.png" alt="">
+      </a>
+      <a href="https://twitter.com/motorbikeexpo?lang=en" class="" target="_blank" rel="noopener noreferrer">
+        <img class="img-ratio mx-3" src="./assets/icons/twitter-icon.png" alt="">
```

Comment:
- [ ] [OPTIONAL] Please kindly add alt text to all images to make your website accessible